pyproject.toml

```diff
@@ -7,7 +7,7 @@ version_files = ["pyproject.toml"]
 
 [tool.ruff.lint]
 select = ['F', 'E', 'W', 'B']
-ignore = ['E501', 'E701']
+ignore = ['E501', 'E701', 'B023']
```

Comment:
This new warning is for unbound variable captures in a for loop. Python really isn't a great language for functional programming.

Something like this:
```python
for i in range(10):
  obj = {}
  call_later(lambda: obj['a'] = i)
```
depending on when that `call_later` function runs (e.g. immediately, or some other time in another thread) the result of the loop might be different.

With the `Maybe` monad, we use exactly this pattern. However, we know that function executions always happen synchronously and are blocking, so this is a safe pattern. So we'll just tell `ruff` to ignore it.